packages/satusehat/configuration-schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "baseUrl": {
+      "title": "Base URL",
+      "type": "string",
+      "description": "Satusehat base URL",
+      "format": "uri",
+      "minLength": 1,
+      "examples": [
+        "https://www.satusehat.com"
+      ]
+    },
+    "clientId": {
+      "title": "Client Id",
+      "type": "string",
+      "minLength": 1,
+      "description": "Your Satusehat client id",
+      "examples": [
+        "the-long-uuid-provided-by-satusehat-for-your-organization"
+      ]
+    },
+    "clientSecret": {
+      "title": "Client Secret",
+      "type": "string",
+      "description": "Client Secret",
+      "minLength": 1,
+      "examples": [
+        "the-long-secret-key-provided-by-satusehat"
+      ]
+    },
+    "access_token": {
+      "title": "Access Token",
+      "type": "string",
+      "description": "Your Satusehat access token",
+      "writeOnly": true,
+      "minLength": 1,
+      "examples": [
+        "the-long-access-token-from-your-auth"
+      ]
+    }
+  },
+  "type": "object",
+  "additionalProperties": true,
+  "required": [
```

Comment:
Ok I'm now worried that if the oauth token is set, clientId and clientSecret are optional.

I'm not actually sure how we want to handle this so I've asked in slack. This is probably fine for v1 but I want to leave this comment here for now.